src/it/scala/uk/gov/hmrc/govukfrontend/support/TemplateIntegrationSpec.scala

```diff
@@ -39,6 +44,8 @@ abstract class TemplateIntegrationSpec[T: OWrites: Arbitrary](govukComponentName
   override def overrideParameters(p: Test.Parameters): Test.Parameters =
     p.withMinSuccessfulTests(20)
 
+  val component = app.injector.instanceOf[C]
```

Comment:
Thinking about naming, is this a `component`, or is a `template`? `component` feels a like imprecise to me.

Reply:
I think we call these things components everywhere else?

Reply:
for me it feels right to use component here, because twirl component in my head is "function that outputs html" rather than the rendered output itself - which I might think of as an example of a design system component